### PR TITLE
Add documentation to resources

### PR DESCRIPTION
**Pull request by SylvainCorlay**
In the resources page (where we find links to mailing list, github org, gitter chat, stack overflow), there was no link to the documentation.

Besides, 
 - I now use the icon that was used for mailing-list for the documentation and use an envelope icon for the mailing list. The envelope icon comes from https://commons.wikimedia.org/wiki/File:Envelope_icon.svg (aside of a change of color and canvas size).
 - The "Jupyter on GitHub" link is now at the top of the list.

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
Before:
before

After:
after

**Comment by minrk** [comment]
👍 from me, but I'll let more public-facing community/documentation people decide.

cc @ellisonbg @fperez @willingc

**Comment by willingc** [comment]
Excellent idea @SylvainCorlay.

**Comment by Carreau** [comment]
+1 the envelop Icon seem to be the wrong shade of gray though.

**Comment by SylvainCorlay** [comment]
@Carreau fixed.

PS: I was actually using a gray that is used in another icon in an invisible layer. Looking at the code for these SVGs, you realize that they are weird. A lot of hidden things, self-intersecting paths... I think that it is very likely that some browser don't render them correctly.

**Comment by SylvainCorlay** [comment]
Merge?

**Comment by Carreau** [comment]
Weird SVG need to be discussed with Brian, and I want to touch word with
him about better technical review of the website in general. Care to open
an issue?

**Comment by SylvainCorlay** [comment]
Sure.

**Comment by fperez** [comment]
Great, thanks! I'm totally +1 on merging this as-is though, even if there are minor issues with the icons/SVG that can be addressed elsewhere. I don't want the perfect to become the enemy of the good in terms of letting us iterate.  

I won't push for an immediate merge right away either, to at least give a day or two for folks to comment.  But if it stalls without further input in that timeframe, I say go ahead and merge, we can always refine later.

**Comment by SylvainCorlay** [comment]
-> from the other PR discussion 

As per discussion earlier with Brian, I find it a bit worrying that (even very minor) PRs involving a UI component triggers so much discussion, and requires so many people to get involved before things get merged. It makes sense for a project like notebook, which is more mature, but not for the website which is much simpler and almost in a "draft" state.

Before we adopt a very formal process for the jupyter.org website, we should get to a more acceptable state. There are tons of easy wins and simple issues that we can fix, and we could fix the most obvious problems by iterating quickly and through drive-by contributions...

**Comment by fperez** [comment]
I actually was in full agreement with you: I only meant to give a chance for feedback from someone else just in case, but I explicitly did *not* want to drag this on for a long time *at all*.  Because I'm similarly worried that I want us to be able to iterate quickly and improve things gradually, rather than blocking on even small fixes for multiple layers of review.

Honestly, I'd be +1 on merging this even in its current state if it had been sitting there for a while. I think for brand new PR that's not even a day old, waiting at least a day just in case someone closer to the repo wants to pitch in is a good practice.

But I really did *not* mean at all to introduce extra delay, believe me: I really want us to treat design as a fluid part of the process that can just as much be iteratively refined, rather than a blocker on all development.

What do you think of the idea of giving it just a day to sit, and merging tomorrow if there isn't further input?

**Comment by SylvainCorlay** [comment]
Sure no problem. This comment was also related to the (just merged) minor css improvements in jupyterlab...

File: community.html [hunk 1]
```diff
@@ -18,38 +18,50 @@
...
-                        <p>A Google Group for general Jupyter discussions of Jupyter's use.</p>
+                        <h2>Jupyter on <br id="github-break">GitHub</h2>
+                        <p>GitHub is our primary place to report issues with Jupyter.</p>
                     </div>
                     <div class="col-md-2 community-button">
-                        <a href="https://groups.google.com/forum/#!forum/jupyter">VIEW</a>
+                        <a href="https://github.com/jupyter/help">VIEW</a>
+                    </div>
+                </div>
+                <div class="col-sm-offset-3 col-sm-6 col-sm-offset-3 col-md-offset-0 col-md-12 community-section">
```
Comment (Carreau) [inline community.html]: why the `col-sm-offset-3`, and  `col-sm-offset-3 col-md-offset-0` ?

Comment (SylvainCorlay) [inline community.html]: this is a repetition of the last  `community-section` for stack overflow, looking at it now.

**Comment by Carreau** [comment]
> As per discussion earlier with Brian, I find it a bit worrying that (even very minor) PRs involving a UI component triggers so much discussion, and requires so many people to get involved before things get merged. It makes sense for a project like notebook, which is more mature, but not for the website which is much simpler and almost in a "draft" state.
> 
> Before we adopt a very formal process for the jupyter.org website, we should get to a more acceptable state. There are tons of easy wins and simple issues that we can fix, and we could fix the most obvious problems by iterating quickly and through drive-by contributions...

I agree that sometime we discuss too much, but at the same time **a lot** of things on jupyter.org were not technically reviewed before, and this lead to a lot of technical debt. 

And it's not because something is in a bad shape that we shouldn't make efforts to make clean contributions. 

This will also conflict with #103 so it would be nice to know how much it will conflict, and which one is the easiest to rebase.

File: community.html (hunk 1 above)
Comment (Carreau) [inline community.html]: Ah, I see it too... well the Stack--overflow section is typically something that shouldn't be this way.

**Comment by SylvainCorlay** [comment]
Yes, I am fine with technical arguments and delays because of the lack of time for swamped potential reviewers.

What I meant is that when it comes to UI changes, we often end up kicking in touch with a "this is design work" or having petty arguments over minor things while the PR fixes something else that is utterly broken. This slows down the iterations.

**Comment by SylvainCorlay** [comment]
@carreau I removed the unnecessary classes that were in the stackoverflow section.

**Comment by Carreau** [comment]
+1 as is, as long as it does not make works on #103 too hard for CalPoly students. @willingc there hasn't been much activity on that in a week, what should we expect ?

**Comment by SylvainCorlay** [comment]
#103 does not seem to conflict with this although it already needs a rebase.

**Comment by willingc** [comment]
@SylvainCorlay There's a small conflict with the names of the navbar items proposed by the CalPoly students and this PR. Overall, I like the change that you are proposing to add Documentation here since the docs have quite a bit of info on connecting to the community.

Here's what the students proposed in #103 (with more screenshots in this [design doc](https://docs.google.com/document/d/1uTPsTF5gPqCE6SdXMmUAC9V3f1FFV0CpzL3xBNWTifo/edit?pref=2&pli=1#heading=h.bqvm4htg1bh3): 

> Visual Changes made:

- Reduced 8 labels to 6 labels
- Removed “Donate” button from navigation bar
- Moved “Install” and “Documentation” to most left
- Renamed “Community” label to “Connect”
- Renamed “Project” label to “About”

@SylvainCorlay If you are cool with this menu being renamed from "Community" to "Connect" in the future, I think that this is a reasonable (and helpful) addition for the end user.

Overall, I think #103 could be revised and rebased after this PR moves forward and is merged.

*Copying students* @charnpreetsingh1985 @spoorthyv (I'm assuming that your time is focused mostly on JupyterLab project this week.)
cc @ellisonbg

**Comment by SylvainCorlay** [comment]
@willingc I have not changed the navbar here.

**Comment by Carreau** [comment]
> Copying students @charnpreetsingh185 @spoorthyv (I'm assuming that your time is focused mostly on JupyterLab project this week.)

Ok, it would be great to know where things are pushed forward to know what we can expect to be work on, review it, or set up for later.

**Comment by SylvainCorlay** [comment]
Guys, there is *no conflict* with this other PR!

**Comment by willingc** [comment]
@SylvainCorlay I realize that the code does not conflict between the two PRs. The only consideration was the proposed new name of the navbar item "Connect" and adding Documentation under it. I think this PR is a good change, and I am +1 (as I was earlier today) on merging this.

This could be merged now and then adjusted in the future if needed :smile:

**Comment by willingc** [comment]
Merging. Others - please feel free to revert or open a new PR to enhance this. Thanks @SylvainCorlay!

**Comment by SylvainCorlay** [comment]
Thanks @willingc we almost had as much discussion on adding a link as for the widget gallery page. This is exactly what I meant earlier...

**Comment by fperez** [comment]
Thanks for merging @willingc! Once we had some extra feedback (which indeed led to one further commit) and it was clear there was no conflict with #103, I'm glad you went ahead with the merge.  

@Carreau's points about technical review are obviously important, as he's gone through the #103 review carefully.  But since this PR had basically *content* with very little design (which can also be polished later if it's not the focus of the PR) and zero low-level CSS work, it's totally sensible to merge once we agree the content is a good idea.

We can keep iterating and improving later on, let's really try hard not lose that mindset :)

**Pull request merged**